### PR TITLE
jackaudio: 1.9.17 -> 1.9.19

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pennae** [comment]
LGTM, tested successfully with network and midi.

File: pkgs/misc/jackaudio/default.nix [hunk 1]
```diff
@@ -27,13 +27,13 @@ let
 in
 stdenv.mkDerivation rec {
   name = "${prefix}jack2-${version}";
```
Comment (SuperSandro2000) [inline pkgs/misc/jackaudio/default.nix:29]:
```suggestion
  pname = "${prefix}jack2";
```

Comment (magnetophon) [reply pkgs/misc/jackaudio/default.nix]: OK, done.

**Comment by SuperSandro2000** [comment]
I think this should go through staging.

**Comment by pennae** [comment]
also closes #134000 #133933 when merged (both duplicates, but to staging)

**Comment by pennae** [comment]
@SuperSandro2000 is it too late to get this (or #134000) merged for 21.11?

**Comment by SuperSandro2000** [comment]
Gotta say I love the GitHub UI. That should have gone to staging but whatever, should be fine I hope.

**Pull request merged**